ml/Config.cc

```diff
@@ -43,6 +43,10 @@ void ml_config_load(ml_config_t *cfg) {
     std::string anomaly_detection_grouping_method = config_get(config_section_ml, "anomaly detection grouping method", "average");
     time_t anomaly_detection_query_duration = config_get_number(config_section_ml, "anomaly detection grouping duration", 5 * 60);
 
+    size_t num_training_threads = config_get_number(config_section_ml, "num training threads", 4);
```

Comment:
is this related to `number of models per dimension` in some way?

Reply:
No, this configures how many threads to use for training, ie. how many CPU cores to have available for training.